src/js/util.js

```diff
@@ -8,18 +8,32 @@ fruskac.Util = (function () {
     Util.prototype = {
 
         /**
-         * Add a class to a DOM element
+         * Add a class to a DOM element if class does not exist
          * @param {HTMLDomElement} element
          * @param {string} className
          */
         addClass: function (element, className) {
+            var self = this;
+            if (self.hasClass(element,className))
+              return;
             if (element.hasOwnProperty('classList'))
                 element.classList.add(className);
             else
                 element.className += ' ' + className;
         },
 
         /**
+         * Check if a DOM element has a class
+         * @param {HTMLDomElement} element
+         * @param {string} className
+         */
+        hasClass: function (element, className) {
+            if ((element.hasOwnProperty('classList') && element.classList.contains(className)) || (' ' + element.className + ' ').indexOf(' ' + className + ' ') > -1)
+              return true;
```

Comment:
This is valid code, but please include curly braces, to maintain code readability.

Reply:
It seemed strange to me that rest of the code was without curlers, but I followed the same pattern. I agree, it will be more readable with braces. I'll add them.